x2text-service/app/controllers/controller.py

```diff
@@ -141,4 +144,12 @@ def process() -> Any:
         )
     x2_text_audit.status = "Failed"
     x2_text_audit.save()
-    return response.text, response.status_code
+    return_val = ""
```

Comment:
@muhammad-ali-e since it might need migrations - we can take it up as a future change. Besides I believe we should try and store the data in tenant schemas rather than the public schema